### PR TITLE
should not fmt.printf in the e2e test

### PR DESCRIPTION
**Pull request by tianshapjq**
**What this PR does / why we need it**:
should not fmt.printf in the e2e test

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
none
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
thanks, is this the only occurrence of `fmt.Print*()` in `e2e`?

/lgtm

**Comment by tianshapjq** [comment]
@nikhita clean finish

**Review by resouer — APPROVED** [review]
The change generally looks good except a nits.

cc @liggitt

File: test/e2e/framework/authorizer_util.go [hunk 1]
```diff
@@ -62,7 +62,7 @@ func WaitForNamedAuthorizationUpdate(c v1beta1authorization.SubjectAccessReviews
 		// GKE doesn't enable the SAR endpoint.  Without this endpoint, we cannot determine if the policy engine
 		// has adjusted as expected.  In this case, simply wait one second and hope it's up to date
 		if apierrors.IsNotFound(err) {
-			fmt.Printf("SubjectAccessReview endpoint is missing\n")
+			glog.Infof("SubjectAccessReview endpoint is missing\n")
```
Comment (resouer) [inline test/e2e/framework/authorizer_util.go]: Since we did not need to format anything here, how about just `glog.Info()`?

Comment (nikhita) [reply test/e2e/framework/authorizer_util.go]: +1

**Comment by nikhita** [comment]
> @nikhita clean finish

@tianshapjq awesome! can you make the changes that @resouer suggested and squash the commits to a single commit?

**Comment by tianshapjq** [comment]
@nikhita done

File: test/e2e/node/events.go [hunk 2]
```diff
@@ -101,7 +100,7 @@ var _ = SIGDescribe("Events", func() {
 				return false, err
 			}
 			if len(events.Items) > 0 {
-				fmt.Println("Saw scheduler event for our pod.")
+				framework.Logf("Saw scheduler event for our pod.")
```
Comment (nikhita) [inline test/e2e/node/events.go]: not relevant to this PR, but maybe we should add `framework.Log` function too.

**Comment by nikhita** [comment]
lgtm from my side. will leave it to @resouer to tag it.

**Comment by tianshapjq** [comment]
@nikhita may I take your time for asking another question? what's the difference between e2e and e2e_node test? I ran the e2e test via `make WAHT=test/e2e/e2e_test.go && go run hack/e2e.go --get=false -- --ginkgo-parallel=3 --provider=none --test --test_args="--ginkgo.focus=***"` and found it only ran the test cases in the e2e dir, how can I build and run test cases in the e2e node? thanks!

**Comment by tianshapjq** [comment]
@resouer can this be approved?

**Comment by tianshapjq** [comment]
@resouer could this lgtm?

**Comment by neolit123** [comment]
thank you for your patience. sorry about that...
/grumble

/lgtm
/sig node
/sig instrumentation

/assign @deads2k @mikedanese @liggitt @sttts

**Pull request merged**